modules/rostests/apitests/atl/CComObject.cpp

```diff
@@ -63,13 +63,13 @@ class CTestObject :
     END_COM_MAP()
 };
 
-
+#ifndef __RATL__ // Avoid confliction of CAtlExeModuleT
```

Comment:
"conflict"
Why is there conflict?

Reply:
There is `CAtlExeModuleT` instance also in `AtlObjMap.cpp`.
This causes assertion failure at `CAtlModule::CAtlModule`.